UPGRADE-6.3.md

```diff
@@ -142,6 +142,19 @@ monolog:
        level: warning
 ```
 
+## Backend Pagedef Configuration
+
+The backend now provides a new sidebar menu which will replace the classic main menu in a future version.
+If your project uses custom pagedef files (`*.pagedef.php`), consider adding the sidebar to these files. This is
```

Comment:
Where are they (normally) stored?

Reply:
In a bundle this is `Resources/BackendPageDefs`. My recommendation is to search for *.pagedef.php though, as indicated in the guide.